molecule/_shared/base.yml

```diff
@@ -30,10 +30,10 @@ provisioner:
   playbooks:
     prepare: ../_shared/prepare.yml
     converge: ../_shared/converge.yml
+    verify: ../_shared/verify.yml
   inventory:
     group_vars:
       consul_instances:
         consul_node_role: bootstrap
 verifier:
-  name: testinfra
-  directory: ../_shared/tests
```

Comment:
I guess that with this PR, the two Python files are obsolete, right? I noticed that in the previous commit you removed the `flake8` verification, which only would have applied to them. If these files are indeed unused, please remove them.

Reply:
Thanks, fixed!